graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodeGenTest.kt

```diff
@@ -5065,4 +5065,40 @@ It takes a title and such.
         ).generate()
         assertThat(result.javaDataTypes[0].typeSpec.fieldSpecs[1].type.toString() == "java.lang.String")
     }
+
```

Comment:
Consider adding a test case where the field that takes an input argument is not at the root, but a nested child.